scripts/shell.py

```diff
@@ -0,0 +1,3 @@
+from app import create_app
+
+create_app()
```

Comment:
`from app.models import *` or something fancier so we can get `User` instead of `user.User` ?